README.md

```diff
@@ -101,52 +185,27 @@ Build and deploy a Maven project with SonarQube analysis and Artifactory deploym
-- Proper Maven settings.xml configuration for Artifactory authentication (provided by the action)
+- The Java and Maven tools must be pre-installed. Use of `mise` is recommended.
+- Required to use the `*.sonarsource.parent:parent` Maven parent.
 
 ### Usage
 
 ```yaml
-name: Build
-on:
-  push:
-    branches:
-      - master
-      - branch-*
-  pull_request:
-  merge_group:
-  workflow_dispatch:
-
-jobs:
-  build:
-    concurrency:
-      group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
-      cancel-in-progress: ${{ github.ref_name != github.event.repository.default_branch }}
-    runs-on: ubuntu-24.04-large
-    name: Build
-    permissions:
-      id-token: write
-      contents: write
-    steps:
-      - uses: actions/checkout@08eba0b27e820071cde6df949e0beb9ba4906955 # v4.3.0
-      - uses: SonarSource/ci-github-actions/build-maven@v1
+permissions:
+  id-token: write
+  contents: write
+steps:
+  - uses: actions/checkout@v5
+  - uses: SonarSource/ci-github-actions/config-maven@v1
+  - uses: SonarSource/ci-github-actions/build-maven@v1
 ```
 
 ### Inputs
 
 | Input                         | Description                                                                                                                | Default                                                              |
 |-------------------------------|----------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------|
-| `public`                      | Whether to build and deploy with/to public repositories                                                                    | Auto-detected from repository visibility                             |
-| `artifactory-reader-role`     | Suffix for the Artifactory reader role in Vault                                                                            | `private-reader` for private repos, `public-reader` for public repos |
 | `artifactory-deployer-role`   | Suffix for the Artifactory deployer role in Vault                                                                          | `qa-deployer` for private repos, `public-deployer` for public repos  |
 | `deploy-pull-request`         | Whether to deploy pull request artifacts                                                                                   | `false`                                                              |
-| `maven-local-repository-path` | Path to the Maven cache directory, relative to the user home directory                                                     | `.m2/repository`                                                     |
-| `maven-opts`                  | Additional Maven options to pass to the build script (`MAVEN_OPTS`)                                                        | `-Xmx1536m -Xms128m`                                                 |
```

Comment:
I'm really not sure about this. To me it does not obviously make sense to set the Maven OPTS in the config-maven action.
This is needed for the build, not the setup of the version. And it is very common to run multiple Maven commands with different OPTS, typically depending on the memory consumption by the tests.